src/dialog.jsx

```diff
@@ -123,58 +122,24 @@ let Dialog = React.createClass({
     resize: '_handleResize',
   },
 
+  getInitialState() {
+    return {
+      muiTheme: this.context.muiTheme ? this.context.muiTheme : ThemeManager.getMuiTheme(DefaultRawTheme),
+    };
+  },
+
   getDefaultProps() {
     return {
       autoDetectWindowHeight: false,
       autoScrollBodyContent: false,
       actions: [],
       repositionOnUpdate: true,
-      defaultOpen: false,
```

Comment:
@oliviertassinari - sorry, not sure I understand what you mean here?  what is it you want me to do?  I'm not sure I see the need for `defaultOpen` at all?